### ISSUE TITLE
configure: error: cannot guess build type; you must specify one in [install.packages("RPostgreSQL")]

### ISSUE THREAD

**Issue by Tom-python0121** [tomoakin/RPostgreSQL#115]
See:
```console
> install.packages("RPostgreSQL")
Installing package into ‘/root/R/aarch64-redhat-linux-gnu-library/4.0’
(as ‘lib’ is unspecified)
trying URL 'https://mirrors.bfsu.edu.cn/CRAN/src/contrib/RPostgreSQL_0.6-2.tar.gz'
Content type 'application/octet-stream' length 579968 bytes (566 KB)
==================================================
downloaded 566 KB

* installing *source* package ‘RPostgreSQL’ ...
** package ‘RPostgreSQL’ successfully unpacked and MD5 sums checked
** using staged installation
checking for gcc... gcc
checking for C compiler default output file name... a.out
checking whether the C compiler works... yes
checking whether we are cross compiling... no
checking for suffix of executables...
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking build system type... src/config.guess: unable to guess system type

This script, last modified 2011-10-19, has failed to recognize
the operating system you are using. It is advised that you
download the most up to date version of the config scripts from

  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD
and
  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD

If the version you run (src/config.guess) is already up to date, please
send the following data and any information you think might be
pertinent to <config-patches@gnu.org> in order to provide the needed
information to handle your system.

config.guess timestamp = 2011-10-19

uname -m = aarch64
uname -r = 4.18.0-193.el8.aarch64
uname -s = Linux
uname -v = #1 SMP Fri May 8 11:05:12 UTC 2020

/usr/bin/uname -p = aarch64
/bin/uname -X     =

hostinfo               =
/bin/universe          =
/usr/bin/arch -k       =
/bin/arch              = aarch64
/usr/bin/oslevel       =
/usr/convex/getsysinfo =

UNAME_MACHINE = aarch64
UNAME_RELEASE = 4.18.0-193.el8.aarch64
UNAME_SYSTEM  = Linux
UNAME_VERSION = #1 SMP Fri May 8 11:05:12 UTC 2020
configure: error: cannot guess build type; you must specify one
ERROR: configuration failed for package ‘RPostgreSQL’
* removing ‘/root/R/aarch64-redhat-linux-gnu-library/4.0/RPostgreSQL’

The downloaded source packages are in
        ‘/tmp/RtmpTKOfdi/downloaded_packages’
Warning message:
In install.packages("RPostgreSQL") :
  installation of package ‘RPostgreSQL’ had non-zero exit status
```
how can I resolve it?

**Comment by Tom-python0121** [comment]
### Information on your system
* **Python:** 3.6.8
* **Platform:** linux-centos8-aarch64
* **Concretizer:** original

**Comment by benz0li** [comment]
See https://github.com/tomoakin/RPostgreSQL/issues/93#issue-261654485: `config.guess` and `config.sub` are outdated and need to be updated.

A patched version of RPostgreSQL 0.6-2 is available at https://gitlab.b-data.ch/benz0li/rpostgresql/-/releases/0.6-2-patched.

To install from source:

```bash
curl -sSL https://gitlab.b-data.ch/benz0li/rpostgresql/-/package_files/6/download -o RPostgreSQL_0.6-2.tar.gz
R CMD INSTALL RPostgreSQL_0.6-2.tar.gz
```
